worlds/satisfactory/Options.py

```diff
@@ -0,0 +1,574 @@
+        "450 coupons (~1,7b points)": 1671112500,
+        "500 coupons (~2b points)": 2294578500,
+        "550 coupons (~3b points)": 3056467000,
+        "600 coupons (~4b points)": 3970650000,
+        "650 coupons (~5b points)": 5051216000,
+        "700 coupons (~6b points)": 6311854500,
+        "750 coupons (~8b points)": 7766437500,
+        "800 coupons (~9b points)": 9429103500,
+        "850 coupons (~11b points)": 11313492000,
+        "900 coupons (~13b points)": 13433475000,
+        "950 coupons (~16b points)": 15803241000,
+        "1000 coupons (~18b points)": 18436379500
+    }
+
+class ResourceSinkPointsPerMinute(NamedRange):
+    """
+    Does nothing if *AWESOME Sink Points (per minute)* goal is not enabled.
+
+    Continuously Sink an amount of items to maintain a sink points per minute of this amount of points for 10 minutes to finish.
+    This setting is in *points per minute* on the orange track, so DNA Capsules don't count.
+    This option's presets are example production thresholds - you don't have to sink exactly those specific items.
+
+    Use the **TFIT - Ficsit Information Tool** mod or the Satisfactory wiki to find out how many points items are worth.
+    """
+    # Coupon data for above comment from https://satisfactory.wiki.gg/wiki/AWESOME_Shop
+    display_name = "AWESOME Sink points per minute"
+    default = 50000
+    range_start = 1000
+    range_end = 10000000
+    special_range_names = {
+        "~500 screw/min": 1000,
+        "~100 reinforced iron plate/min": 12000,
+        "~100 stator/min": 24000,
+        "~100 modular frame/min": 40000,
+        "~100 smart plating/min": 50000,
+        "~20 crystal oscillator/min": 60000,
+        "~50 motor/min": 76000,
+        "~10 heavy modular frame/min": 100000,
+        "~10 radio control unit": 300000,
+        "~10 fused modular frame/min": 625000,
+        "~10 supercomputer/min": 1000000,
+        "~10 pressure conversion cube/min": 2500000,
+        "~10 nuclear pasta/min": 5000000,
+        "~4 ballistic warp drive/min": 10000000,
+    }
+
+class HardDriveProgressionLimit(Range):
+    """
+    How many Hard Drives can contain progression items.
+    Hard Drives above this count cannot contain progression, but can still be Useful.
+    
+    There are 118 total hard drives in the world and the current implementation supports up to 100 progression hard drives.
+    """
+    display_name = "Hard Drive Progression Items"
+    default = 20
+    range_start = 0
+    range_end = 100
+
+class FreeSampleEquipment(Range):
+    """
+    How many free sample items of Equipment items should be given when they are unlocked.
+    
+    (ex. Jetpack, Rifle)
+    """
+    display_name = "Free Samples: Equipment"
+    default = 1
+    range_start = 0
+    range_end = 10
+
+class FreeSampleBuildings(Range):
+    """
+    How many copies of a Building's construction cost to give as a free sample when they are unlocked.
+    Space Elevator is always excluded.
```

Comment:
```suggestion
    How many copies of a Building's construction cost are given as a free sample when they are unlocked.
    Space Elevator is always excluded.
```
This one is just for consistency